src/format.js

```diff
@@ -19,6 +33,12 @@ const formatSummary = status => {
 	return summaryBits.join( ', ' );
 };
 
+/**
+ * Format a string that compares a previous an current code changesets.
```

Comment:
This compares the previous lint result against the current one to work out which errors have been fixed; I actually don't think this is used any more maybe?

```suggestion
 * Format a string that compares previous and current check runs.
```